.github/ISSUE_TEMPLATE/depr-ticket.yml

```diff
@@ -81,18 +81,11 @@ body:
       description: If you plan to mark the code for deprecation, explain how.
     validations:
       required: false
-  - type: textarea
-    id: migration
-    attributes:
-      label: Migration
-      description: If automated migration will be needed, explain your migration plan.
-    validations:
-      required: false
   - type: textarea
     id: addl-info
     attributes:
       label: Additional Info
-      description: If there is any additional publicly sharable information or data from your earlier analysis, include that.
+      description: If there is any additional publicly sharable information or data from your earlier analysis, include that. Will there need to be any manual database migrations needed, for example?
```

Comment:
I think it would be great to ask for a documented transition plan somewhere in the template. Maybe under the description for "Replacement"? I think this new sentence, if we keep it, is a possible detail of a transition plan and could move to wherever we document that.
- Additionally, "document transition plan" could be an example task in the first task list field.

Reply:
I can't get lost in this now either. Maybe I'll come back to this in July. Thanks for moving things ahead. One step at a time.